Source/Core/CMakeLists.txt

```diff
@@ -9,6 +9,4 @@ add_subdirectory(UICommon)
 add_subdirectory(VideoCommon)
 add_subdirectory(VideoBackends)
 
-if(ENABLE_QT2)
-  add_subdirectory(DolphinQt2)
-endif()
+add_subdirectory(DolphinQt2)
```

Comment:
Was this meant to be made unconditional?

Reply:
Yes, as you can hopefully see it clearly states '**** TEMPORARY HACK ***' I do not intend to keep this commit around.